packages/components/src/Banner/Banner.mdx

```diff
@@ -20,7 +20,15 @@ import { Banner } from "@jobber/components/Banner";
 ```
 
 <Playground>
-  <Banner type="success">You've successfully Hulked out</Banner>
+  <Banner
+    type="success"
+    primaryAction={{
+      label: "Poke Me",
+      onClick: () => alert("💥 SMASH 💥")
+    }}
+  >
+    You've successfully Hulked out
+  </Banner>
```

Comment:
I wonder if it is worth being more explicit in the docs here about using this pattern, vs. putting buttons in as child components.

Reply:
It might also be worth it to have a dedicated example section for our new case, instead of just adding to the default. You could talk in the new section about the button props and what that means (see the Modal component for examples).